libs/internal/jarvis-routes/src/lib/internal-jarvis-routes.tsx

```diff
@@ -0,0 +1,33 @@
+import { Pagination } from '@tal-test/pagination';
```

Comment:
this should be in a folder called internal, not internal in the name

Reply:
The file name was autogenerated, as said in other comments, I was focusing something else, let me know if you want me to change it to another name.